target/linux/ramips/dts/mt7620a_dlink_dir-510l.dts

```diff
@@ -85,7 +86,9 @@
 			};
 
 			partition@210000 {
-				compatible = "amit,jimage";
+				compatible = "openwrt,uimage", "denx,uimage";
+				openwrt,ih-magic = <IH_MAGIC_OKLI>;
+				openwrt,offset = <0x10000>;
```

Comment:
the property `openwrt,offset` is relative to start of eraseblock, not start of flash

in other words, used when the kernel starts in the middle of a block

so you can remove the property for all of them, as long as kernel is aligned to start of a block

Reply:
If I remove this line, it stop working:

```
[    1.449721] spi-nor spi0.0: mx25l12805d (16384 Kbytes)
[    1.460080] 3 fixed-partitions partitions found on MTD device spi0.0
[    1.472751] Creating 3 MTD partitions on "spi0.0":
[    1.482309] 0x000000000000-0x000000010000 : "jboot"
[    1.493049] 0x000000010000-0x000000ff0000 : "firmware"
[    1.506788] 0x000000ff0000-0x000001000000 : "config"
[    1.929031] /dev/root: Can't open blockdev
[    1.937288] VFS: Cannot open root device "(null)" or unknown-block(0,0): error -6
[    1.952213] Please append a correct "root=" boot option; here are the available partitions:
[    1.968872] 1f00              64 mtdblock0
[    1.968876]  (driver?)
[    1.981907] 1f01           16256 mtdblock1
[    1.981910]  (driver?)
[    1.994928] 1f02              64 mtdblock2
[    1.994930]  (driver?)
[    2.007947] Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0)
```

Reply:
ah ok, nevermind

maybe without offset property, it looks for rootfs where the loader is